libs/ledger-live-common/src/wallet-api/logic.ts

```diff
@@ -194,3 +194,37 @@ export function signMessageLogic(
 
   return uiNavigation(account, formattedMessage);
 }
+
+export const bitcoinFamillyAccountGetXPubLogic = (
+  { manifest, accounts, tracking }: WalletAPIContext,
+  walletAccountId: string
+): Promise<string> => {
+  tracking.bitcoinFamillyAccountXpubRequested(manifest);
```

Comment:
I know there is no spec for wallet API & tracking, but I think tracking is mostly interesting when it's an actual step in the user journey (CQFD all blocking flows). Here I feel that we are more on sentry territory.